botorch/utils/multi_objective/scalarization.py

```diff
@@ -32,13 +32,15 @@ def get_chebyshev_scalarization(
     Augmented Chebyshev scalarization:
         objective(y) = min(w * y) + alpha * sum(w * y)
 
-    Outcomes are first normalized to [0,1] for maximization (or [-1,0] for minimization)
+    Outcomes are multiplied by -1, then normalized to [0,1] for maximization (or [-1,0] for minimization)
     and then an augmented Chebyshev scalarization is applied.
 
+    Since typically the augmented chebyshev scalarization is minimized, we multiply the resulting quantity by -1.
```

Comment:
```suggestion
    Outcomes are multiplied by -1 (since botorch by default assumes maximization
    of the underlying outcomes), then normalized to [0,1] for maximization (or [-1,0]
    for minimization) and then an augmented Chebyshev scalarization is applied.

    Since typically the augmented Chebyshev scalarization is minimized, we
    multiply the resulting quantity by -1.
```